src/utils.js

```diff
@@ -72,33 +101,70 @@ export default class Utils {
    * @param {HTML Element} val2 - the html element to add to or remove from
    * @param {HTML Element} val3 - the html element to add to or remove from
    */
-  help(type, val1, val2, val3) {
+  static help(type, val1, val2, val3) {
     if (type === 'add') {
-      this.tog('add', 'd-none', val1);
-      this.tog('rem', 'd-none', val2);
-      this.tog('add', 'strike', val3);
-      this.change(val1.dataset.id, true);
+      Utils.tog('add', 'd-none', val1);
+      Utils.tog('rem', 'd-none', val2);
+      Utils.tog('add', 'strike', val3);
+      Utils.change(val1.dataset.id, true);
     }
     if (type === 'rem') {
-      this.tog('rem', 'd-none', val1);
-      this.tog('add', 'd-none', val2);
-      this.tog('rem', 'strike', val3);
-      this.change(val1.dataset.id, false);
+      Utils.tog('rem', 'd-none', val1);
+      Utils.tog('add', 'd-none', val2);
+      Utils.tog('rem', 'strike', val3);
+      Utils.change(val1.dataset.id, false);
     }
   }
 
   /**
    * @function load - the load function
    * @param {Array} tasks - The tasks array
    */
-  load(tasks) {
+  static load(tasks) {
     document.querySelectorAll('li.list').forEach((elem) => {
       const task = tasks.find((item) => item.index === parseInt(elem.children[0].dataset.id, 10));
       if (task.completed) {
         const child = elem.children;
-        this.help('add', child[0], child[1], child[2]);
-        child[1].addEventListener('click', () => this.help('rem', child[0], child[1], child[2]));
+        Utils.help('add', child[0], child[1], child[2].firstElementChild);
+        child[1].addEventListener('click', () => Utils.help('rem', child[0], child[1], child[2].firstElementChild));
+      }
+    });
+  }
+
+  /**
+   * @function edit - Edits the todo and push changes to localStorage
+   */
+  static edit() {
+    document.querySelectorAll('.list form').forEach((form) => {
+      form.addEventListener('submit', (e) => {
+        e.preventDefault();
+        Utils.change(e.target.dataset.id, e.target.firstElementChild.value);
+      });
+    });
+  }
```

Comment:
- [ ] I think this function does not provide an accessible experience since the user is expected to hit the key `enter` upon changing some of the tasks. Both, users that do not use the keyboard as much or users in tactile devices will have a hard time changing the tasks.

Instead, I think you could use the 'change' event, which registers anytime a input from a form changes. [You can read more about it here.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event)

Reply:
Ok. Thanks @D4nielJ , I will look into it now.